packages/react-conformance/src/utils/mountWithCleanup.ts

```diff
@@ -0,0 +1,29 @@
+import { mount as enzymeMount, ReactWrapper } from 'enzyme';
+
+let wrapper: ReactWrapper | undefined;
+
+/**
+ * A wrapper around enzyme's mount that helps with unmounting and cleanup
+ * This is an approach that's adopted by quite a few users of enzyme.
+ * https://github.com/enzymejs/enzyme/issues/911
+ * @returns Enzyme wrapper
+ */
+export const mount = (...args: Parameters<typeof enzymeMount>) => {
+  const enzymeWrapper = enzymeMount(...args);
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  wrapper = enzymeWrapper;
```

Comment:
Why it's not assignable?

Reply:
enzyme `mount` is generic, which takes the props of the component passed in. SInce the `wrapper` is never exported or returned I decided not to recreate all those generic types

Reply:
Can you cast instead of ts-ignore so it's less "nuclear"?